server/sonar-web/src/main/js/app/components/GlobalFooter.tsx

```diff
@@ -17,27 +17,27 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-// @flow
-import React from 'react';
+import * as React from 'react';
 import { Link } from 'react-router';
 import GlobalFooterSonarCloud from './GlobalFooterSonarCloud';
 import GlobalFooterBranding from './GlobalFooterBranding';
 import { translate, translateWithParameters } from '../../helpers/l10n';
 
-/*::
-type Props = {
-  hideLoggedInInfo?: boolean,
-  productionDatabase: boolean,
-  onSonarCloud?: { value: string },
-  sonarqubeVersion?: string
-};
-*/
+interface Props {
+  hideLoggedInInfo?: boolean;
+  productionDatabase: boolean;
+  onSonarCloud?: { value: string };
+  sonarqubeVersion?: string;
+}
 
-export default function GlobalFooter(
-  { hideLoggedInInfo, productionDatabase, onSonarCloud, sonarqubeVersion } /*: Props */
-) {
+export default function GlobalFooter({
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') The function GlobalFooter has a cyclomatic complexity of 12 which is higher than the threshold of 10 [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS1541)